pkg/gadgets/capabilities/gadget.go

```diff
@@ -0,0 +1,133 @@
+// Copyright 2019-2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package capabilities
+
+import (
+	"encoding/json"
+	"fmt"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/kinvolk/inspektor-gadget/pkg/gadgets"
+	"github.com/kinvolk/inspektor-gadget/pkg/gadgets/capabilities/tracer"
+
+	standardtracer "github.com/kinvolk/inspektor-gadget/pkg/gadgets/capabilities/tracer/standard"
+	"github.com/kinvolk/inspektor-gadget/pkg/gadgets/capabilities/types"
+
+	gadgetv1alpha1 "github.com/kinvolk/inspektor-gadget/pkg/apis/gadget/v1alpha1"
+)
+
+type Trace struct {
+	resolver gadgets.Resolver
+
+	started bool
+	tracer  tracer.Tracer
+}
+
+type TraceFactory struct {
+	gadgets.BaseFactory
+}
+
+func NewFactory() gadgets.TraceFactory {
+	return &TraceFactory{
+		BaseFactory: gadgets.BaseFactory{DeleteTrace: deleteTrace},
+	}
+}
+
+func (f *TraceFactory) Description() string {
+	return `capabilities traces open() system calls`
```

Comment:
```suggestion
	return `capabilities suggests security capabilities for the securityContext`
```

Reply:
I took the opportunity to update it to `Trace security capability checks`.